examples/ros/opt.ros.distro.lib.demo_nodes_cpp

```diff
@@ -0,0 +1,12 @@
+#include <tunables/global>
+#include <tunables/ros>
+
+profile demo_nodes_cpp/talker @{ROS_INSTALL_LIB}/demo_nodes_cpp/talker {
```

Comment:
Looking at this now, I wonder what has changed wrt. AppArmor as for the executable permission for executable path to no longer be necessary, as compared to previous profiles I've tested. Is this due to an upstream change in an abstraction I've imported?